.vscode/settings.json

```diff
@@ -1,8 +1,5 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.formatOnSave": true,
-  "jest.jestCommandLine": "npm run test:repository -- --watch --env=jsdom",
-  "jest.autoRun": {
```

Comment:
leave the onSave option ;) I use it

Reply:
How do I override it for me? I find it annoying

Reply:
nvm my local settings override it